ballerina/client.bal

```diff
@@ -176,21 +176,24 @@ public isolated client class Client {
         }
     }
 
-    # Creates an object.
+    # Creates an object. 
     #
     # + bucketName - The name of the bucket
     # + objectName - The name of the object
     # + payload - The file content that needed to be added to the bucket
     # + cannedACL - The access control list of the new object
     # + objectCreationHeaders - Optional headers for the `createObject` function
+    # + userMetadataHeaders - Optional headers to add user defined meta data
```

Comment:
```suggestion
    # + userMetadataHeaders - Optional headers to add user-defined metadata
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-aws.s3/pull/109/commits/c954e9c7e0252bb7cfd9acd6d696d155e241169c